### DISCUSSION TITLE
Additional database field for circumcision status

### DISCUSSION THREAD

**Discussion by mountainmantra** [stashapp/stash#3265] [Ideas]
The current implementation of the performer table is gender specific, for females. It seems that the original intention of the Measurements field is for Breast Size, with the following field being Fake Tits. 

I suggest either breaking out the gender specific fields so they are only added based on selection of gender, or the addition of Circumcision as a field so that circumcision status can be tracked along with Penis length in the Measurements field.
<img width="1380" alt="Screen Shot 2022-12-10 at 4 43 52 PM" src="https://user-images.githubusercontent.com/11653895/206876677-261cdd54-0d33-40d6-89ab-33938322002b.png">

**Comment by KrisVardo** [comment]
I came here to suggest this, also.  I was thinking it should be "Penis Type" because that could allow for circumcision status, as well as be open-ended for other situations (prosthetic, etc.).

I think it should be a new field, rather than overload "Fake Tits".  There are trans performers with both.  Both "Penis Type" and "Fake Tits" should probably be available always, and can just be "n/a" if they don't apply.

I think it's Ok to overload Measurements for any kind of body measurements that are relevant to that performer.

**Comment by KrisVardo** [comment]
I opened a feature request for this here:
https://github.com/stashapp/stash/issues/3267
